qiita_db/exceptions.py

```diff
@@ -72,6 +72,14 @@ def __init__(self, repeated_headers):
                      "header(s): %s." % ', '.join(repeated_headers),)
 
 
+class QiitaDBDuplicateSamplesError(QiitaDBError):
+    """Exception for error when a MetadataTemplate has duplicate columns"""
+    def __init__(self, repeated_samples):
+        super(QiitaDBDuplicateSamplesError, self).__init__()
+        self.args = ("Duplicate samples found in MetadataTemplate. Repeated "
```

Comment:
Can remove the "Repeated samples:" from this to save room, since it's saying the same thing as the first sentence.